mysql-test/suite/sys_vars/r/num_sharded_listen_sockets_basic.result

```diff
@@ -1,13 +1,13 @@
 SELECT @@GLOBAL.num_sharded_listen_sockets;
 @@GLOBAL.num_sharded_listen_sockets
-1
+4
 1 Expected
```

Comment:
Do we need to update this line in the test file?  Should it be "4 expected"?  And at line 11 and 24.